cmd/dep/init_test.go

```diff
@@ -144,3 +147,125 @@ const Qux = "yo yo!"
 	// vendor should have been created & populated
 	h.MustExist(h.Path("src/" + root + "/vendor/github.com/Sirupsen/logrus"))
 }
+
+// TestCaseInsentitive is test for Windows. This should work even though set
+// difference letter cases in GOPATH.
+func TestCaseInsentitive(t *testing.T) {
```

Comment:
I was thinking that this test should have two purposes. The first is checking that error doesn't occor with init command. The second is checking that dep should work with status command. Both should be tested with CaSeInSeNtIvE GOPATH. So below is enough to check. How do you think?

```go
func TestCaseInsentitive(t *testing.T) {
	if runtime.GOOS != "windows" {
		t.Skip("skip this test on non-Windows")
	}
	test.NeedsExternalNetwork(t)
	test.NeedsGit(t)

	h := test.NewHelper(t)
	defer h.Cleanup()

	h.TempDir("src")

	// Shuffle letter case
	rs := []rune(strings.ToLower(h.Path(".")))
	for i, r := range rs {
		if unicode.IsLower(r) {
			rs[i] = unicode.ToUpper(r)
		} else {
			rs[i] = unicode.ToLower(r)
		}
	}
	h.Setenv("GOPATH", string(rs))

	h.RunGo("get", "github.com/pkg/errors")
	repoDir := h.Path("src/github.com/pkg/errors")
	h.RunGit(repoDir, "checkout", "v0.8.0")

	m := `package bar

import (
	"io"
	"github.com/pkg/errors"
)

var err = errors.Wrap(io.EOF, "EOF")
`
	h.TempFile("src/github.com/golang/notexist/bar.go", m)
	h.Cd(h.Path("src/github.com/golang/notexist"))

	h.Run("init")
	h.Run("status")
}
```